src/webgpu/api/validation/createSampler.spec.ts

```diff
@@ -0,0 +1,19 @@
+export const description = `
+createSampler validation tests.
+
+Test Plan: 
+
+* lodMinClamp:
+  * < 0
+  * = NAN
+
+* lodMaxClamp:
+  * > MAX_SAFE_INTEGER
+  * = NAN
+  * =,> FLT_MAX
+  * < lodMinClamp
```

Comment:
Will need to test this for several values of lodMinClamp. I would recommend combining these two tests into one, where the cases are combinations of lodMinClamp and lodMaxClamp, e.g.

```ts
.params([
  { lodMinClamp: -1 },
  { lodMinClamp: 0 },
  /*...*/
  { lodMaxClamp: Number.MAX_VALUE },
  { lodMaxClamp: 0 },
  /*...*/
  { lodMinClamp: 2, lodMaxClamp: 2 },
  { lodMinClamp: 2.01, lodMaxClamp: 2 },
])
```